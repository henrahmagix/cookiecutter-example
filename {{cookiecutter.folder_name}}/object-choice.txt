Objects are not supported.

Output: {{cookiecutter.object_of_options}}
