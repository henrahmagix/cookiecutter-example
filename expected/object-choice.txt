Objects are not supported.

Output: OrderedDict([('a', 'The letter A'), ('b', 'The letter B'), ('c', 'The letter C')])
